pkg/operator/ceph/cluster/predicate.go

```diff
@@ -43,6 +60,13 @@ func predicateForNodeWatcher(ctx context.Context, client client.Client, context
 		},
 
 		UpdateFunc: func(e event.UpdateEvent) bool {
+			objOld := e.ObjectOld.(*corev1.Node)
+			objNew := e.ObjectNew.(*corev1.Node)
```

Comment:
What happens if either of these casting operations fail?

I think we need to check, to do our due diligence:

```go
obj, ok := object.(*typecast)
```

Reply:
Updated